### ISSUE TITLE
Mod breaks open to LAN

### ISSUE THREAD

**Issue by SystemlessDev** [MustangXPress7/Cyan-Warrior-Swords-Minecraft-Mod-#9]
**Minecraft Version:** 1.19.3
**Forge Version:** 44.1.21
**Mod Version:** 3.4.1

**Steps to recreate:** 
- Add `Cyan Warrior Swords Mod` to your mods folder. Make two instances of the game. 
- Make a world on client 1 and open it to LAN.
- Join the world with client 2 (you might have to use direct connect with `localhost:port`).
- See the error below on your screen.
![image](https://github.com/MustangXPress7/Cyan-Warrior-Swords-Minecraft-Mod-/assets/45434286/b9b534ca-ae97-4555-839d-fa5e9cbd86e9)

**Relevant lines in the host's log:**
```
[20:23:51] [Render thread/INFO] [minecraft/ChatComponent]: [System] [CHAT] Systemless_ joined the game
[20:23:51] [Server thread/INFO] [minecraft/ServerGamePacketListenerImpl]: Systemless_ lost connection: Internal Exception: io.netty.handler.codec.EncoderException: java.lang.NullPointerException: Cannot invoke "net.minecraft.resources.ResourceLocation.toString()" because "p_130086_" is null
```

**Full log:**
https://paste.essential.gg/6AqdPB2QRE

**Comment by EmiGITs** [comment]
Hi SystemlessDev, thanks for reporting this issue! I'll look into it. Are you using the latest version from curse?

**Comment by SystemlessDev** [comment]
I am indeed using the latest version (3.4.1) from curseforge, as stated in the original report ;) 
Specifically this version https://www.curseforge.com/minecraft/mc-mods/cyan-warrior-swords-mod/files/4558799

**Comment by EmiGITs** [comment]
Hi! We uploaded a new version to curse with a fix for this and updated the mod to 1.19.4 with the fix included. Can you test it yourself to confirm it is fixed?

**Comment by SystemlessDev** [comment]
It seems good to go now! :) Thank you! :D
